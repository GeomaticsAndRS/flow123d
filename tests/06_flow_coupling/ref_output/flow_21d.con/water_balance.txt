	"time"	"region"	"quantity [m(3)]"	"flux"	"flux_in"	"flux_out"	"mass"	"source"	"source_in"	"source_out"	"integrated_flux"	"integrated_source"	"error"
	0	"channel"	"water_volume"	0	0	0	0	-50	-50	0	0	0	0
	0	"plane"	"water_volume"	0	0	0	0	0	0	0	0	0	0
	0	".channel"	"water_volume"	1.67347e-10	0	1.67347e-10	0	0	0	0	0	0	0
	0	".plane"	"water_volume"	-50	-50	1.65716e-07	0	0	0	0	0	0	0
