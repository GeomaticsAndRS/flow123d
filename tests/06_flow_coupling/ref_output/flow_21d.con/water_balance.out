# ---------------------------------------- WATER BALANCE ----------------------------------------
# Time of computed water balance: inf


# Boundary water balance:
# [boundary_id] [label]                  [total_balance][total_outflow][total_inflow][time]        
# -----------------------------------------------------------------------------------------------
  4             .channel                   7.4433e-08    7.4433e-08    0             inf           
  2             .plane                     -50           2.93716e-06   -50           inf           
# -----------------------------------------------------------------------------------------------
# total boundary balance                   -50           3.0116e-06    -50           


# Source fluxes over material subdomains:
# [region_id]   [label]                  [total_balance]                            [time]        
# -----------------------------------------------------------------------------------------------
  3             channel                    -50                                       inf           
  1             plane                      0                                         inf           
# -----------------------------------------------------------------------------------------------
# total sources balance                    -50           


